src/Integration/src/IntegrationBase/Util/IntegrationContextUtils.cs

```diff
@@ -19,6 +19,10 @@ public static class IntegrationContextUtils
 
         public const string INTEGRATION_SIMPLE_EVALUATION_CONTEXT_BEAN_NAME = "integrationSimpleEvaluationContext";
 
+        public const string MESSAGE_HANDLER_FACTORY_BEAN_NAME = "integrationMessageHandlerMethodFactory";
```

Comment:
Is `BEAN` the best term we can use here?

Reply:
I am open to suggestions but there is a lot of them in this code base - not sure its worth the effort. @dtillman ?

Reply:
SERVICE could be used as well if you don't like BEAN...